fe/fe-core/src/main/java/com/starrocks/catalog/Type.java

```diff
@@ -807,7 +832,7 @@ public boolean supportBloomFilter() {
     }
 
     public static final String NOT_SUPPORT_JOIN_ERROR_MSG =
-            "Type (nested) percentile/hll/bitmap/json/struct/map not support join";
+            "Type (nested) percentile/hll/bitmap/json not support join";
```

Comment:
why not support json btw

Reply:
as the serialized results of json columns are not stable @Seaven